packages/web-react/src/types/shared/element.ts

```diff
@@ -1,12 +1,19 @@
-import { DetailedHTMLProps, HTMLAttributes } from 'react';
+import { DetailedHTMLProps, HTMLAttributes, HTMLProps } from 'react';
 import { OmittedUnsafeStyleProps, StyleProps } from './style';
 
+/** Returns all relevant attributes and their types from a given HTML Element */
 export type SpiritDetailedHTMLProps<E> = DetailedHTMLProps<HTMLAttributes<E>, E>;
+/** Returns all relevant attributes and their types, combined basic and detailed, from a given HTML Element */
+export type SpiritCombinedHTMLProps<E> = DetailedHTMLProps<HTMLAttributes<E>, E> & HTMLProps<E>;
 
 export type SpiritDivElementBaseProps = SpiritDetailedHTMLProps<HTMLDivElement>;
 export type SpiritDialogElementBaseProps = SpiritDetailedHTMLProps<HTMLDialogElement>;
 export type SpiritElementBaseProps = SpiritDetailedHTMLProps<HTMLElement>;
+export type SpiritInputElementBaseProps = SpiritCombinedHTMLProps<HTMLInputElement>;
```

Comment:
This is not the right place for these types. `inputs` will be better.

Reply:
I disagree, This is the right place because there are other elements defined in that file and they are defined in the same way, so I thought it would be appropriate to put them here. According to this logic, would we then also divide the `HTMLDivElement` into a separate `types/shared/div.ts`? Likewise for dialogue? Here are simply the basic props of the elements in which we remove the style and className and, on the contrary, add our UNSAFE. I don't see any reason to put this in `inputs.ts`, where only props of the input are dealt with.

Reply:
But I renamed them and moved them instead to inputs.ts

Reply:
🆗